.gitignore

```diff
@@ -62,3 +62,6 @@ typings/
 
 # Macbook
 .DS_Store
+
+#Security
```

Comment:
Great that you are taking care of the security aspect 😄 Can you please message me the contents of that file so that I can have it locally? Otherwise I can't run your app... You can message me on gitter :)

Reply:
It's on Gitter attached in our group chat 😉